### PR TITLE
Test Pod for cpu shared + exclusive apps

### PR DESCRIPTION
**Pull request by bnshr**

### CODE REVIEW

**Review by sebrandon1 — COMMENTED** [review]
Looks good but I think should add a script to deploy it similar to something like:
https://github.com/test-network-function/cnf-certification-test-partner/blob/main/scripts/deploy-test-pods.sh

This will build the environment vars like `TNF_EXAMPLE_CNF_NAMESPACE` for you.

**Comment by bnshr** [comment]
> 

When do you call this script?

File: scripts/deploy-test-pods.sh [hunk 1]
```diff
@@ -6,7 +6,16 @@ source "$SCRIPT_DIR"/init-env.sh
 
 mkdir -p ./temp
 cat ./test-target/local-pod-under-test.yaml | APP="testdp" RESOURCE_TYPE="Deployment" MULTUS_ANNOTATION=$MULTUS_ANNOTATION "$SCRIPT_DIR"/mo > ./temp/rendered-local-pod-under-test-template.yaml
+
+cp ./test-target/cpu-exclusive-pool-rt-sched-policy-pod.yaml /temp/rendered-cpu-exclusive-pool-rt-sched-policy-pod.yaml
+cp ./test-target/cpu-shared-pool-non-rt-sched-policy-pod.yaml /temp/rendered-cpu-shared-pool-non-rt-sched-policy-pod.yaml
```
Comment (sebrandon1) [inline scripts/deploy-test-pods.sh:11]: So the real "magic" in the above rendered templates is the "$SCRIPT_DIR"/mo

#/ Mo is a mustache template rendering software written in bash.  It inserts
#/ environment variables into templates.

It'll fill in the {{ }} blocks with variables.

Comment (sebrandon1) [reply scripts/deploy-test-pods.sh:11]: `cp ./test-target/cpu-exclusive-pool-rt-sched-policy-pod.yaml /temp/rendered-cpu-exclusive-pool-rt-sched-policy-pod.yaml`

Should be something like:

`cat ./test-target/cpu-exclusive-pool-rt-sched-policy-pod.yaml | "$SCRIPT_DIR"/mo > ./temp/rendered-cpu-exclusive-pool-rt-sched-policy-pod.yaml`

**Review by sebrandon1 — APPROVED** [review]

**Review by sebrandon1 — APPROVED** [review]

**Pull request merged**